docs/specs/component.md

```diff
@@ -208,11 +208,14 @@ implement since errors are specific to the component.
 
 * Properties
   * `error` - The specifics of the error condition, such as system error code, etc.
```

Comment:
Should we drop this property too given https://github.com/vectordotdev/vector/pull/9205/commits/89f6974de69a4370c0505e0ec69d30d1e31548e8 ? Or perhaps other metrics would have it.

Reply:
No, the property is still used by the log, just not the metric.